h3i/src/prompts/h3/mod.rs

```diff
@@ -423,6 +429,23 @@ fn prompt_extension() -> InquireResult<Action> {
     Ok(action)
 }
 
+pub fn prompt_connection_close() -> InquireResult<Action> {
+    let is_app = prompt_yes_no("application close:")?;
+    let error_code = prompt_varint("error code:")?;
```

Comment:
There's already a transport error code suggester in `prompt_close_stream` that we should reuse.

Reply:
I wonder if we should refactor all of that out into a separate `errors.rs` file